tools/walletextension/test/wallet_extension_test.go

```diff
@@ -295,8 +295,11 @@ func TestGetStorageAtForReturningUserID(t *testing.T) {
 	respJoin := makeHTTPEthJSONReqWithPath(walletHTTPPort, "v1/join")
 	userID := string(respJoin)
 
+	specialAddressToGetUserID := "0x0000000000000000000000000000000000000000"
+	randomAddress := "0x123ABC0000000000000000000000000000000000"
```

Comment:
Why use`specialAddressToGetUserID` instead of a string ?

Reply:
see the answer above + decided to make a variable and not hardcode it in all cases separately